proto/zetachain/zetacore/emissions/params.proto

```diff
@@ -7,6 +7,27 @@ option go_package = "github.com/zeta-chain/node/x/emissions/types";
 
 // Params defines the parameters for the module.
 message Params {
+  option (gogoproto.goproto_stringer) = false;
+  string validator_emission_percentage = 5;
```

Comment:
Can you provide a comment this a sample value here? Is it "1", or "1.0" or "0.01", etc...